1-js/02-first-steps/01-hello-world/article.md

```diff
@@ -1,87 +1,87 @@
+Der `<script>`-Tag hat einige Attribute die heutzutage nur selten Verwendung finden, aber noch immer in alten Code vorhanden sind:
 
-The `type` attribute: <code>&lt;script <u>type</u>=...&gt;</code>
-: The old HTML standard, HTML4, required a script to have a `type`. Usually it was `type="text/javascript"`. It's not required anymore. Also, the modern HTML standard totally changed the meaning of this attribute. Now, it can be used for JavaScript modules. But that's an advanced topic; we'll talk about modules in another part of the tutorial.
+Das `type`-Attribute: <code>&lt;script <u>type</u>=...&gt;</code>
+: Der alte HTML-Standard, HTML4, setzte bei einem Skript den `Typ` voraus. Üblicherweise war es `type="text/javascript"`. Dies wird nicht länger benötigt. Zudem, der moderne HTML-Standard hat die Bedeutung des Attributs völlig verändert. Jetzt kann es für JavaScript-Module verwendet werden. Aber dies ist ein weiterführendes Thema; wir werden über Module in einem anderen Teil des Tutorials sprechen.
 
-The `language` attribute: <code>&lt;script <u>language</u>=...&gt;</code>
-: This attribute was meant to show the language of the script. This attribute no longer makes sense because JavaScript is the default language. There is no need to use it.
+Das `language`-Attribute: <code>&lt;script <u>language</u>=...&gt;</code>
+: Dieses Attribute war gedacht, um die Sprache des Skripts anzuzeigen. Dieses Attribute macht nicht länger Sinn, weil JavaScript die Standard-Sprache ist. Es gibt keinen Grund es zu nutzen.
 
-Comments before and after scripts.
-: In really ancient books and guides, you may find comments inside `<script>` tags, like this:
+Kommentare vor und nach Skripten.
+: In wirklich alten Büchern und Anleitungen findest du vielleicht, innerhalb von `<script>`-Tags, Kommentare wie diese:
 
     ```html no-beautify
     <script type="text/javascript"><!--
         ...
     //--></script>
     ```
 
-    This trick isn't used in modern JavaScript. These comments hide JavaScript code from old browsers that didn't know how to process the `<script>` tag. Since browsers released in the last 15 years don't have this issue, this kind of comment can help you identify really old code.
+    Dieser Trick wird im modernen JavaScript nicht verwendet. Diese Kommentare verstecken JavaScript-Code für alte Browser, die nicht wissen wie der `<script>`-Tag verarbeitet wird. Da Browser-Veröffentlichungen der letzten 15 Jahre dieses Problem nicht haben, kann dir diese Art von Kommentar helfen wirklich alten Code zu identifizieren.
 
 
-## External scripts
+## Externe Skripte
 
-If we have a lot of JavaScript code, we can put it into a separate file.
+Wenn wir viel JavaScript-Code haben, können wir diesen in eine separate Datei legen.
 
-Script files are attached to HTML with the `src` attribute:
+Skript-Dateien werden zu HTML mit dem `src`-Attribute hinzugefügt:
 
 ```html
-<script src="/path/to/script.js"></script>
+<script src="/pfad/zum/skript.js"></script>
```

Comment:
```suggestion
<script src="/pfad/zum/script.js"></script>
```

Reply:
Good point! Please update this.. Then we're ready to merge 💃

Reply:
I am not allowed to update this, right? The autho mustr, right? Or can I, and just didn't realize?

Reply:
@JoeGab Christian has to implement the changes... Then we can review again and merge if everything is fine 👍. The Authomation doesn't mark this as Requested changes becauase you aren't a contributor rn.